azerothcore/azerothcore-wotlk
# fix(Core/Deps): Update recastnavigation to last version

thanks @jackpoz for pointing this out

-----------------

## How to test:

- Checkout this branch and recompile, including recompiling the client data files extractor tools (maps/dbc/vmaps/mmaps )
- Extract the data files (use a clean client) and run a server with the newly extracted data (enable vmaps/mmaps)
- Log in the game and have a look around to make sure the maps work fine. You could check, for example, stuff like elevators, pathing, casting spells from behind walls, etc...
